mungegithub/github/github.go

```diff
@@ -315,6 +315,20 @@ func (config *Config) AddRootFlags(cmd *cobra.Command) {
 // PreExecute will initialize the Config. It MUST be run before the config
 // may be used to get information from Github
 func (config *Config) PreExecute() error {
+	glog.Infof("token: %#v\n", config.Token)
```

Comment:
Maybe in a follow up you can convert the config struct to yaml and print that. Harder for people to forget to add a line here when they add an option?